src/EventProducerService.cpp

```diff
@@ -39,15 +39,32 @@ void EventProducerService::setProducedEvents()
 
 byte EventProducerService::createDefaultEvent(byte evValue)
 {
+  unsigned int eventNum = 0;
+  unsigned int nodeNum = module_config->nodeNum;
   byte data[4];
```

Comment:
Please move this declaration of data down to before line 58 where it is first used.
Maybe also rename this to "nn_en" to make it clearer what it is for.

Reply:
I assume you only mean the declaration at line 43 to be moved as eventNum is used at line 48.
These are two separate variables.  Not sure what you mean by renaminh to "nn_en"

Reply:
No, i meant "data" on line 44. "data" is a generic name so want to call it "nn_en".